routers/api/v1/repo/pull_review.go

```diff
@@ -539,3 +539,79 @@ func prepareSingleReview(ctx *context.APIContext) (*models.Review, *models.PullR
 
 	return review, pr, false
 }
+
+// DismissPullReview dismiss a review for a pull request
+func DismissPullReview(ctx *context.APIContext, opts api.DismissPullReviewOptions) {
```

Comment:
Potentially should be able to undismiss

Reply:
It's reasonable, I will add api to do it. But it's hard to show it on ui, because dismissd reviw won't be shown on ui.